src/eko/output/manipulate.py

```diff
@@ -61,6 +61,8 @@ def xgrid_reshape(
         )
         target_rot = b.get_interpolation(targetgrid.raw)
         eko.rotations._targetgrid = targetgrid
+        # update metadata
+        eko.update_metadata({"rotations": {"_targetgrid": targetgrid}})
```

Comment:
Are you sure you need the underscore?

I know the attribute is that one, the other being a property, but it'd look nicer without, for manual inspection.
At the end of the day, the initial underscore discriminate between the virtual property, and the actual representation, but when we serialize there is only the representation and nothing more.

Reply:
The point is that at the end the metadata file should be read and used to construct a `Rotations` object. In order to do that it expects `_targetgrid` and similar.

Reply:
Indeed, I was proposing to change what `Rotations` expects, but this we can discuss together, it's easier :)